go/prometheusexporter/collector/collector.go

```diff
@@ -223,9 +303,26 @@ func (c *Collector) Update(stream *util.Stream) {
 	if c.httpRequestSummaryVec == nil {
 		c.registerHttpRequestSummaryVec()
 	}
+	for _, m := range c.opts.Resources.TimeAndCallResources {
```

Comment:
From the metric name `"logjam:action:" + metric + "_distribution_seconds"` (e.g. `"logjam:action:db_calls_distribution_seconds"` one would assume that the metrics measure a duration. While this is true for `TimeResources` it is not true for `CallResources` as they represent a count of calls, don't they?

For `CallResources` i'd not offer a summary, but a simple counter. If you think a histogram makes sense (e.g. to see how many requests require less than 10 DB calls to be executed and how many requests need more than 100 DB calls to be executed) we should have a different set of buckets for those call metrics as well.

Reply:
Yes call resources are not measured in seconds. My mistake.

I'll change them to be _totals and counters.

Alas, this will everything more complex again.

Reply:
Updated. @0robustus1 can you check again?